<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InvestmentPage</class>
 <widget class="QWidget" name="InvestmentPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>846</width>
    <height>284</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Investment Wizard Page</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,50,1">
   <item>
    <spacer name="leftSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Please select the appropriate columns to use, corresponding to your data.
If there is just a single security in the file, enter its name below.
Otherwise, select the column for the symbol, and for the description or name.</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout" columnstretch="0,0,0,0,0,0,0">
       <item row="0" column="0" colspan="7">
        <widget class="Line" name="line_3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="labelInv_memo">
         <property name="accessibleName">
          <string comment="column containing memo field"/>
         </property>
         <property name="text">
          <string>Memo</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLabel" name="labelInv_quantity">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="accessibleName">
          <string comment="column containing quantity field"/>
         </property>
         <property name="text">
          <string>Quantity</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QComboBox" name="m_amountCol">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Select column containing amount field.</string>
         </property>
         <property name="maxVisibleItems">
          <number>12</number>
         </property>
        </widget>
       </item>
       <item row="1" column="5">
        <widget class="QComboBox" name="m_priceFraction">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>If necessary, select a fraction/multiplier for compatibility
between imported and stored prices. For instance, if the
import price is in cents but the file is priced in dollars, select 0.01.</string>
         </property>
         <property name="maxVisibleItems">
          <number>5</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="labelInv_date">
         <property name="toolTip">
          <string comment="verb, Select column containing date field">Select column containing date field</string>
         </property>
         <property name="accessibleName">
          <string comment="column containing date field"/>
         </property>
         <property name="text">
          <string>Date</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="labelInv_price">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="accessibleName">
          <string comment="column containing price field"/>
         </property>
         <property name="text">
          <string>Price</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="m_dateCol">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Select column containing date field.</string>
         </property>
         <property name="maxVisibleItems">
          <number>12</number>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QLabel" name="labelInv_amount">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string comment="column for monetary value">Amount</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QComboBox" name="m_quantityCol">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Select column containing quantity field.</string>
         </property>
         <property name="maxVisibleItems">
          <number>12</number>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QComboBox" name="m_priceCol">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Select column containing price field.</string>
         </property>
         <property name="maxVisibleItems">
          <number>12</number>
         </property>
        </widget>
       </item>
       <item row="2" column="5">
        <widget class="QComboBox" name="m_symbolCol">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>If more than one stock is present in the file,
and if the stock symbols are shown in one of the columns,
select it here.</string>
         </property>
        </widget>
       </item>
       <item row="3" column="6">
        <widget class="QPushButton" name="m_clear">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Clear selected column entries.</string>
         </property>
         <property name="text">
          <string comment="verb, to clear selected columns">Clear</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0" colspan="7">
        <widget class="Line" name="line_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="QLabel" name="labelInv_priceFraction">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="accessibleName">
          <string comment="noun, conversion factor to align pricing in the imported file with KMM's internal prices."/>
         </property>
         <property name="text">
          <string comment="noun, conversion fraction/multiplier to use to make compatible the csv pricing with KMM pricing.">Price Fraction</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="4">
        <widget class="QLabel" name="labelInv_symbolCol">
         <property name="text">
          <string comment="@label:listbox stock symbol">Symbol</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="5">
        <widget class="QComboBox" name="m_nameCol">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>If a descriptive column also contains an activity type, select it here.
See also Filter text field.</string>
         </property>
        </widget>
       </item>
       <item row="3" column="4">
        <widget class="QLabel" name="labelInv_nameCol">
         <property name="text">
          <string>Name</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="7" column="2">
        <widget class="QLabel" name="labelInv_feeIsPercentage">
         <property name="text">
          <string>Fee is Percentage</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="labelInv_feeCol">
         <property name="accessibleName">
          <string comment="column containing fee field"/>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string comment="column containing fee field">Fee Column</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="7" column="3" alignment="Qt::AlignRight">
        <widget class="QCheckBox" name="m_feeIsPercentage">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Check box if any fee is a percentage figure rather than a value.</string>
         </property>
         <property name="layoutDirection">
          <enum>Qt::RightToLeft</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="m_typeCol">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Select column containing investment type field.</string>
         </property>
         <property name="maxVisibleItems">
          <number>12</number>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QComboBox" name="m_feeCol">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>If necessary, select column containing fee.
Caution. This might already have been
incorporated into the price.</string>
         </property>
         <property name="maxVisibleItems">
          <number>12</number>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QLineEdit" name="m_feeRate">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="labelInv_feeRate">
         <property name="text">
          <string>Fee rate</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="8" column="3">
        <widget class="QLineEdit" name="m_minFee">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="labelInv_type">
         <property name="accessibleName">
          <string comment="column containing investment type/action field"/>
         </property>
         <property name="text">
          <string comment="column containing investment type/action field">Type/Action</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QComboBox" name="m_memoCol">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Select column containing memo field.
More than one column may be entered successively.
May also be selected to contain a copy of the type or name column.</string>
         </property>
         <property name="maxVisibleItems">
          <number>12</number>
         </property>
        </widget>
       </item>
       <item row="8" column="2">
        <widget class="QLabel" name="labelInv_feeMinValue">
         <property name="text">
          <string>Minimal Fee</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="6" column="0" colspan="7">
        <widget class="Line" name="line">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="7" column="6">
        <widget class="QPushButton" name="m_clearFee">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Clear Fee</string>
         </property>
        </widget>
       </item>
       <item row="8" column="6">
        <widget class="QPushButton" name="m_calculateFee">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Calculate Fee</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="rightSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>m_dateCol</tabstop>
  <tabstop>m_typeCol</tabstop>
  <tabstop>m_memoCol</tabstop>
  <tabstop>m_priceCol</tabstop>
  <tabstop>m_quantityCol</tabstop>
  <tabstop>m_amountCol</tabstop>
  <tabstop>m_priceFraction</tabstop>
  <tabstop>m_symbolCol</tabstop>
  <tabstop>m_nameCol</tabstop>
  <tabstop>m_feeCol</tabstop>
  <tabstop>m_feeIsPercentage</tabstop>
  <tabstop>m_feeRate</tabstop>
  <tabstop>m_minFee</tabstop>
  <tabstop>m_calculateFee</tabstop>
  <tabstop>m_clear</tabstop>
  <tabstop>m_clearFee</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
